src/components/badge/bl-badge.stories.mdx

```diff
@@ -0,0 +1,106 @@
+import { html } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { styleMap } from 'lit/directives/style-map.js';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { Meta, Canvas, ArgsTable, Story, Preview, Source } from '@storybook/addon-docs';
+
+<Meta
+  title="Components/Badge"
+  component="bl-badge"
+  argTypes={{
+    icon: {
+        control: 'text'
+    },
+    size: {
+        options: ['large', 'medium', 'small'],
+        control: {type: 'select'}
+    }
+  }}
+/>
+
+export const BadgeTemplate = (args) => html`<bl-badge
+icon="${ifDefined(args.icon)}"
+size=${ifDefined(args.size)}
+style=${ifDefined(args.styles ? styleMap(args.styles) : undefined)}
+>${unsafeHTML(args.content)}</bl-badge>`
+
+export const VariantTemplate = (args) => html`
+${BadgeTemplate({ styles: { '--bl-badge-bg-color' : 'rgb(254, 242, 232)', '--bl-badge-color': 'rgb(242, 122, 26)' },...args})}
+${BadgeTemplate({ styles: { '--bl-badge-bg-color' : 'rgba(231, 249, 239, 1)', '--bl-badge-color': 'rgba(11, 193, 92, 1)' },...args})}
+${BadgeTemplate({ styles: { '--bl-badge-bg-color' : '#FFF8E6', '--bl-badge-color': '#FFB600' },...args})}
+${BadgeTemplate({ styles: { '--bl-badge-bg-color' : 'var(--bl-color-danger-background)', '--bl-badge-color': 'var(--bl-color-danger)' },...args})}
+${BadgeTemplate({ styles: { '--bl-badge-bg-color' : 'var(--bl-color-alternative-background)', '--bl-badge-color': 'var(--bl-color-alternative)' },...args})}
+${BadgeTemplate({ styles: { '--bl-badge-bg-color' : 'var(--bl-color-featured-background)', '--bl-badge-color': 'var(--bl-color-featured)' },...args})}
+`;
+
+export const WithIconTemplate = (args) => html`
+${BadgeTemplate({ icon: 'info', styles: { '--bl-badge-bg-color' : 'var(--bl-color-accent-primary-background)', '--bl-badge-color': 'var(--bl-color-primary)' },...args})}
+${BadgeTemplate({ icon: 'info', styles: { '--bl-badge-bg-color' : 'var(--bl-color-success-background)', '--bl-badge-color': 'var(--bl-color-success)' },...args})}
+${BadgeTemplate({ icon: 'info', styles: { '--bl-badge-bg-color' : 'var(--bl-color-warning-background)', '--bl-badge-color': 'var(--bl-color-warning)' },...args})}
+${BadgeTemplate({ icon: 'info', styles: { '--bl-badge-bg-color' : 'var(--bl-color-danger-background)', '--bl-badge-color': 'var(--bl-color-danger)' },...args})}
+${BadgeTemplate({ icon: 'info', styles: { '--bl-badge-bg-color' : 'var(--bl-color-alternative-background)', '--bl-badge-color': 'var(--bl-color-alternative)' },...args})}
+${BadgeTemplate({ icon: 'info', styles: { '--bl-badge-bg-color' : 'var(--bl-color-featured-background)', '--bl-badge-color': 'var(--bl-color-featured)' },...args})}
+`;
+
+export const SizeTemplate = (args) => html`
+${BadgeTemplate({ size:'small', ...args })}
+${BadgeTemplate({ size:'medium', ...args })}
+${BadgeTemplate({ size:'large', ...args })}
+`;
+
+
+# Badge
+The badge can keep different types of content such as a number, status or icon.
```

Comment:
What does "number" mean here? Is there a use case example for this?

Reply:
No this is not a use case I meant to say we can keep any content number, text, icon etc. If this is confusing, we can delete it.